src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -290,6 +291,8 @@ public function deleteAction(int $combinationId): JsonResponse
     }
 
     /**
+     * @todo: this might not be needed anymore
```

Comment:
Actually this will come in handy when we improve bulk deletion and perform them chunk by chunk